src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -467,7 +467,7 @@ def _deploy_arm_template_at_resource_group(cli_ctx,
     validation_result = mgmt_client.validate(resource_group_name=resource_group_name, deployment_name=deployment_name, properties=deployment_properties)
 
     if validation_result and validation_result.error:
-        raise CLIError(validation_result.error)
+        raise CLIError(todict(validation_result.error))
```

Comment:
>todict [](start = 23, length = 6)

can you give a sample of the old/new error output?

Reply:
I added it to the PR description